MIGRATION.md

```diff
@@ -0,0 +1,264 @@
+# Migration Guide
+
+Xrpl.js 3.0 reduces bundle size by 60% and completely removes the need for polyfills for browser applications. It does this with minimal breaking changes. It accomplished this by replacing node specific dependencies with ones which would work in the browser.
+
+The primary breaking change is updating all `Buffer` types to `Uint8Array` since `Buffer` is not supported in the browser. `Buffer` objects can usually be directly replaced with `UInt8Array` as `Buffer` extends `Uint8Array`. The primary difference in functionality is that `Buffer` has additional helper functions. We've listed all functions affected by this change below in the `Uint8Array` section.
```

Comment:
In Xrpl.js 3.0, we've made significant improvements that result in a 60% reduction in bundle size for browser applications. We've also eliminated the need for polyfills with minimal disruption to existing code. This was achieved by replacing node-specific dependencies with ones that are compatible with browsers.

The main change you'll notice is the update of all Buffer types to Uint8Array because browsers don't support Buffer. Fortunately, this transition is relatively straightforward, as Buffer can often be replaced directly with UInt8Array, as Buffer extends Uint8Array. The main functional difference is that Buffer has additional helper functions, which we've listed in the Uint8Array section below for your reference.
```suggestion
Xrpl.js 3.0 reduces bundle size by 60% and completely removes the need for polyfills for browser applications. It does this with minimal breaking changes. It accomplished this by replacing node specific dependencies with ones which would work in the browser.

The primary breaking change is updating all `Buffer` types to `Uint8Array` since `Buffer` is not supported in the browser. `Buffer` objects can usually be directly replaced with `UInt8Array` as `Buffer` extends `Uint8Array`. The primary difference in functionality is that `Buffer` has additional helper functions. We've listed all functions affected by this change below in the `Uint8Array` section.
```